pkgs/development/python-modules/hdate/default.nix

```
@@ -0,0 +1,54 @@
+{ lib
+, astral
+, buildPythonPackage
+, fetchFromGitHub
+, poetry-core
+, pytestCheckHook
+, pythonOlder
+, pytz
+}:
+
+buildPythonPackage rec {
+  pname = "hdate";
+  version = "0.10.2";
+  disabled = pythonOlder "3.6";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "py-libhdate";
+    repo = "py-libhdate";
+    rev = "v${version}";
+    sha256 = "07b0c7q8w6flj4q72v58d3wymsxfp5qz8z97qhhc2977mjx5fsxd";
+  };
+
+  nativeBuildInputs = [
+    poetry-core
+  ];
+
+  propagatedBuildInputs = [
+    astral
+    pytz
+  ];
+
+  checkInputs = [
+    pytestCheckHook
+  ];
+
+  postPatch = ''
+    substituteInPlace pyproject.toml --replace "^2020.5" "^2021"
+  '';
+
+  disabledTests = [
+    # Don't test the README
+    "README.rst"
+  ];
```

That sounds odd, why not

```suggestion
  pytestFlagsArray = [
    "tests"
   ];
```

---

There is doctest which is picked-up.

---

Then that should be in the comment. But in general, why not ask pytest to look at the tests folder only?

---

Updated